### PR TITLE
babel: add page

### PR DESCRIPTION
**Pull request by pxgamer**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/babel.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# babel
+
+> A compiler for writing next generation JavaScript.
```
Comment (agnivade) [inline pages/common/babel.md]: Shouldn't it be transpile ?

Comment (pxgamer) [reply pages/common/babel.md]: Yes, I think.

Comment (sbrl) [reply pages/common/babel.md]: Yes. From [here](https://www.stevefenton.co.uk/2012/11/compiling-vs-transpiling/):

> Compiling
>  - is the general term for taking source code written in one language and transforming into another
> 
> Transpiling
>  - is a specific term for taking source code written in one language and transforming into another language that has a similar level of abstraction

File: pages/common/babel.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- Transpile the input file every time it is changed:
+
+`babel {{path/to/input_file}} --watch`
+
+- Transpile a whole directory of files:
+
+`babel {{path/to/input_directory}}`
+
+- Ignore specified comma-separated files in a directory:
```
Comment (sbrl) [inline pages/common/babel.md]: Can this contain globs?

File: pages/common/babel.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# babel
+
+> A transpiler for writing next generation JavaScript.
```
Comment (agnivade) [inline pages/common/babel.md]: I think the language can be improved as - `A transpiler which converts code from JavaScript ES6/ES7 syntax to ES5 syntax.`. Although it supports other targets too. But as the most general case, this is what it is used for.

File: pages/common/babel.md (hunk 2 above)
Comment (pxgamer) [reply pages/common/babel.md]: Yes, it can contain globs.

**Pull request merged**